### DISCUSSION TITLE
Making the placeholder [enchant] work in the scavenge

### DISCUSSION THREAD

**Discussion by FrancyMak** [Zrips/CMI#7410] [Enhancement]
Hi, I wanted to ask if it would be possible to add the placeholder [enchant] in the message `&eEnchantment worth: &7[price]` so that the player would know which enchant the price refers to.

Because currently the name of the book is "Enchanted Book" in the description it is not specified whether it refers to sharpness or unbreaking for example, so if the two enchantments have different cost, players can't understand "Ok indestructibility costs me 2000, but sharpness costs 5000, so I better take unbreaking from another item".

**Comment by CloudeLecaw** [comment]
Well i like the idea. but please just add it optionaly.

well normaly you define a enchantworth for good one and low enchantworth for bad once. What is good and what is bad if depending on how the servers was setup.
Normaly the players should learn what a high price enchant might be and what a lower enchantworth might should be and what it worth. This worth thing was made to have different extraprices later for the final scavange price, not for seeing the name of the single enchantments that got scavanged.


At least currently they see all enchants they might get and see a final scavange price.
they cant choose to get just one or two. they take all or nothing. And what they get is allready listed on the item they want to scavange..But Yes..i can understand that many players just play to casual these days to be willing to learn the differences of worthless and worthfull and just wanne see the name for each scavange book even if they cant change the final price. 

_**So for the name i would recomment an extra line where it optionaly can be shown if wished. (may by a placeholder. or could be cleared out if not wanted.(so we can still use the same look like it is now))**_

but at least.. hand up for a nice idea for scavenge extansion! Thank you FrancyMak :)
